packages/nativescript-aptabase/index.android.ts

```diff
@@ -1,11 +1,29 @@
-import { AptabaseCommon } from "./common";
+import { Utils } from '@nativescript/core';
+import { AptabaseCommon } from './common';
+import { InitOptions } from '.';
+
+function initOptionsFromOptions(options: InitOptions) {
+	const initOptions = new com.aptabase.InitOptions();
+	if (options?.host) {
+		initOptions.setHost(options.host);
+	}
+
+	return initOptions;
+}
 
 export class Aptabase extends AptabaseCommon {
-    static initialize(appKey: string) {
-        console.log('Not supported on Android yet.')
-    }
-
-    static track(name: string, properties?: any) {
-        console.log('Not supported on Android yet.')
-    }
-}
\ No newline at end of file
+	static initialize(appKey: string, opts?: InitOptions) {
+		const initOptions = initOptionsFromOptions(opts);
+		com.aptabase.Aptabase.getInstance().initialize(Utils.android.getApplicationContext(), appKey, initOptions);
+	}
+
+	static track(name: string, properties?: any) {
```

Comment:
tiny improvement here, we could replace `any` with `Record<string, string | number | boolean>`, that's what I use on aptabase-js